docs/resources/job.md

```diff
@@ -84,6 +93,16 @@ resource "databricks_job" "this" {
       main_class_name = "com.acme.data.Main"
     }
   }
+
+  task {
+    task_key = "c"
+
+    job_cluster_key = "j"
```

Comment:
Could you add cluster key to other tasks in the same job?

Reply:
It's left this way for a reason. I wanted users to see that there can be different approaches of pointing to a cluster. So ane task is pointing to existing, one is running on shared_job cluster and one has "private" job cluster.

Changelog updated